src/odemis/driver/test/tfsbc_test.py

```diff
@@ -60,6 +67,23 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def test_dependency_scanner(self):
+        """Test dependency on scanner."""
+        # TODO only works with xt simulator now - how to make generic? for xt simulator has special flag "sim"
+        if TEST_NOHW:
+            self.microscope = xt_client.SEM(**CONFIG_MB_SEM)
+            for child in self.microscope.children.value:
+                if child.name == CONFIG_MB_SCANNER["name"]:
+                    self.scanner = child
+
+            dependencies = {"scanner": self.scanner}
+            self.bc = tfsbc.BeamShiftController("DC Offset", None, PORT, "FT2OMDD5", dependencies)
+            # Note: for simulator matrix is only 0s
+            calib_md = self.bc.getMetadata()[model.MD_CALIB]
+            self.assertIsNotNone(calib_md)
+            self.assertIsInstance(calib_md, list)
```

Comment:
I think it's fine to only have `self.assertIsInstance(calib_md, list)`, if it's a list it cannot be None.